go-sms_inbound.js

```diff
@@ -1363,6 +1363,8 @@ go.app = function() {
             switch (user_first_word) {
                 case "STOP":
                     return self.states.create("state_find_identity");
+                case "BABY":
```

Comment:
That was the only word in the ticket, I will confirm this and all the other replies.